internal/config/config.go

```diff
@@ -34,22 +35,37 @@ const (
 	ClientTokenEnvVar    = "EQUINIX_API_TOKEN"
 	ClientTimeoutEnvVar  = "EQUINIX_API_TIMEOUT"
 	MetalAuthTokenEnvVar = "METAL_AUTH_TOKEN"
+
+	// This is the initial env var for HTTP logging for backwards
+	// compatibility.  We could now introduce service-specific
+	// variables, if desired, for more granular control of logging
+	HTTPLoggingEnvVar = "TF_LOG"
+
+	// Subsystems for API client logging
+	packetClientSubsystem  = "Equinix Metal (packngo)"
+	equinixClientSubsystem = "Equinix"
+	metalClientSubsystem   = "Equinix Metal (metal-go)"
```

Comment:
Looks like we missed a spot in the SDK conversion.  This could either be `(equinix-sdk-go)` to reflect the previous convention of tagging with the SDK name or it could be `(metalv1)` to align with the `fabricClientSubsystem` format, in which case perhaps we could remove `packetClientSubsystem`, since the logs would already include User-Agent to see which requests came from which SDK.

```suggestion
	metalClientSubsystem   = "Equinix Metal (equinix-sdk-go)"
```